### PR TITLE
Evm path support

### PR DESCRIPTION
**Pull request by chcmedeiros**

### CODE REVIEW

File: app/Makefile [hunk 1]
```diff
@@ -65,35 +66,18 @@ $(error ICONNAME is not set)
 endif
 
 include $(CURDIR)/../deps/ledger-zxlib/makefiles/Makefile.platform
-LDFLAGS += -z muldefs
-CFLAGS += -Wvla
 
-DEFINES += HAVE_HASH HAVE_BLAKE2 HAVE_SHA256 HAVE_SHA512
```
Comment (ftheirs) [inline app/Makefile:71]: I think that this line shouldn't be removed

File: app/Makefile [hunk 2]
```diff
@@ -41,6 +41,7 @@ ifeq ($(COIN),ATOM)
 # Main app configuration
 APPNAME = "Cosmos"
 APPPATH = "44'/118'"
+APPPATH2 = "44'/60'"
```
Comment (ftheirs) [inline app/Makefile]: APPPATH variable will be used by the installer script, I think it's better to keep this format: 
`APPPATH = "44'/118'" --path "44'/60'"
`

File: app/src/apdu_handler.c [hunk 3]
```diff
@@ -33,6 +33,83 @@
...
+
+    const uint8_t payloadType = G_io_apdu_buffer[OFFSET_PAYLOAD_TYPE];
+
+    if (rx < OFFSET_DATA) {
+        THROW(APDU_CODE_WRONG_LENGTH);
+    }
+
+    uint32_t added;
+    switch (payloadType) {
+        case 0:
```
Comment (ftheirs) [inline app/src/apdu_handler.c]: we can replace the numbers from cases with P1_INIT, P1_ADD and P1_LAST

File: app/src/parser.c [hunk 4]
```diff
@@ -183,11 +183,11 @@ __Z_INLINE parser_error_t parser_formatAmountItem(uint16_t amountToken,
         return parser_unexpected_field;
     }
 
-    if (!parser_areEqual(amountToken + 1u, "amount")) {
+    if (!parser_areEqual(amountToken + 1u, (char*) "amount")) {
```
Comment (ftheirs) [inline app/src/parser.c]: you can change in parser_areEqual the input argument to `const char *expected` and then you don't need to cast the string

**Comment by jleni** [comment]
How do we pass this path guidelines failure?

**Comment by ftheirs** [comment]
It checks the app params against this DB: https://github.com/LedgerHQ/ledger-app-database/pull/26/files
So we had to add ETH path

**Review by ftheirs — APPROVED** [review]

**Pull request merged**